### PR TITLE
Fixed #34768 -- Avoided initializing colorama on non-Windows platforms.

### PR DESCRIPTION
**Pull request by sulabhkatila**
Fixed #34768 -- Added platform check before installing colorama.

### CODE REVIEW

File: django/core/management/color.py [hunk 1]
```diff
@@ -4,18 +4,22 @@
...
 import functools
 import os
+import platform
 import sys
 
 from django.utils import termcolors
 
-try:
-    import colorama
+if platform.system() == "Windows":
```
Comment (felixxm) [inline django/core/management/color.py:12]: We normally use `sys.platform == "win32"`:
```suggestion
# Don't initialize colorama on non-Windows platforms.
if sys.platform == "win32":
```

**Review by felixxm — COMMENTED** [review]
@sulabhkatila Thanks for this patch :+1: 

@yourcelf Does it work for you?

**Comment by yourcelf** [comment]
> @yourcelf Does it work for you?

This approach solves my problem. However, I suggest you take a look at colorama's [`just_fix_windows_console`](https://github.com/tartley/colorama/blob/master/colorama/initialise.py#L72) initializer as an alternative to `colorama.init()`.  In addition to checking for `sys.platform == "win32"`, that initializer also handles newer windows' native ANSI support.

Using that initializer would leave the feature and platform detection up to colorama, rather than maintaining them separately here.  The downside is having less clarity as to whether `HAS_COLORAMA` is `True` or `False`, so that may require further tweaks to `supports_color()`.

File: django/core/management/color.py (hunk 1 above)
Comment (sulabhkatila) [reply django/core/management/color.py:12]: I will make the changes.

File: django/core/management/color.py [hunk 2]
```diff
@@ -48,7 +47,11 @@ def vt_codes_enabled_in_windows_registry():
 
     return is_a_tty and (
         sys.platform != "win32"
-        or HAS_COLORAMA
+        or (
+            colorama.fixed_windows_console
+            if hasattr(colorama, "fixed_windows_console")
+            else False
```
Comment (felixxm) [inline django/core/management/color.py:53]: This will crash when `colorama` is not installed, so I restored `HAS_COLORAMA`.

**Comment by felixxm** [comment]
@yourcelf Thanks :+1: Can you check the newest version?

@MinchinWeb @carltongibson Can any of you confirm this works on Windows? (with and without `colorama`) :gift:

**Review by carltongibson — COMMENTED** [review]
This requires `colorama>=0.4.6` (i.e. the latest version) to work ([CHANGELOG](https://github.com/tartley/colorama/blob/master/CHANGELOG.rst)) but it does work (Windows CMD) with that in place. 

👍

**Comment by felixxm** [comment]
> This requires `colorama>=0.4.6` (i.e. the latest version) to work ([CHANGELOG](https://github.com/tartley/colorama/blob/master/CHANGELOG.rst)) but it does work (Windows CMD) with that in place.
> 
> 👍

Thanks for checking :+1: I bumped the minimum supported version.

**Comment by felixxm** [comment]
@sulabhkatila Welcome aboard :boat:

**Comment by MinchinWeb** [comment]
Glad this could get checked and merged so fast!

**Pull request merged**